_episodes/05-starting-jekyll.md

```diff
@@ -7,75 +7,351 @@ questions:
+> From [Jekyll's website](https://jekyllrb.com/docs/front-matter/):
 >
-> Grad student Davis wants to use their current institution name throughout their personal website
-> but will soon be leaving to become a postdoc at another institution.
-> They have created a variable `current_institution` in the YAML header of the `index.md` pages on their site.
-> What changes would they make to use this variable on other pages as well?
-> How might they need to change the following text on their `index.md` page after those changes?
+>   >   Any file that contains a YAML front matter block will be processed by Jekyll as a special file. The front matter must be the first thing in the file and must take the form of valid YAML set between triple-dashed lines.
+{: .testimonial}
+
+Here is an example:
+    
+    ---
+    layout: post
+    title: "My first blog post"
+    author: "Danger Mouse"
+    ---
+    
+Between these triple-dashed lines, you can redefine predefined variables (like `page.layout` or `page.title`) or create custom ones you need locally on the page (like `page.author`). These variables will then be available for you to access using Liquid's tags {% raw %}{{{% endraw %} and {% raw %}}}{% endraw %} further down in the file and also in any files that include this one.                                                                                                                     
+
+> ## Exercise
 >
-> ~~~
-> Davis currently works at {% raw %}{{ page.current_institution }}{% endraw %}.
-> ~~~
-> {: .source}
+> Let's practice making and using local variables. Think of a local variable you may want to use only in your `about.md` or `index.md` page.
+> If you cannot think of any, create a local variable called 'lesson-example' with the value 
+> of 'https://carpentries.github.io/lesson-example/' and reference it in your `index.md`. 
+>
+> What did you add to your `index.md` to create this variable?
+> Where did you add the front matter in your `index.md`?
+> How did you reference that variable?
 >
 > > ## Solution
 > >
-> > They need to move the variable to the `_config.yml`
-> > Then they would change the index code to the following.
-> > ~~~
-> > Davis currently works at {% raw %}{{ site.current_institution }}{% endraw %}.
-> > ~~~
-> > {: .output}
+> > Create a YAML header at the very top of `index.md` and add the `lesson-example` variable in between the
+> > triple-dash delimiters. Your file should now look like:
+> > 
+> >    ~~~
+> >    ---  
+> >    lesson-example: "https://carpentries.github.io/lesson-example/"
+> >    ---  
+> >   
+> >    # {% raw %}{{ site.title }}{% endraw %}
+> >   
+> >    ## Description
+> >    {% raw %}{{ site.description }}{% endraw %}
+> >
+> >    The project is led by {% raw %}{{ site.team_members[0]["name"] }}{% endraw %}.
+> >    [See our full team](/about#team)
+> >
+> >    Have any questions about what we do? [We'd love to hear from you!]({% raw %}mailto:{{ site.email }}{% endraw %})
+> >    ~~~
```

Comment:
not sure we need this repeat of the file again.  might be enough to show only the changed version below

Reply:
I am not sure myself what the best policy is here - showing or highlighting changes would be really good from the instructor perspective. Learners may prefer the whole file.